extension_explorer/views.py

```diff
@@ -87,9 +87,14 @@ def lang_home(lang):
     return render_template('home.html', lang=lang)
 
 
-@app.route('/<lang>/documentation/')
-def documentation(lang):
-    return render_template('documentation.html', lang=lang)
+@app.route('/<lang>/publisher_guidance/')
+def publisher_guidance(lang):
+    return render_template('publisher_guidance.html', lang=lang)
+
+
+@app.route('/<lang>/user_guidance/')
```

Comment:
Ideally, rename the templates to match.

```suggestion
@app.route('/<lang>/publishers/')
def publisher_guidance(lang):
    return render_template('publisher_guidance.html', lang=lang)


@app.route('/<lang>/users/')
```

Reply:
Templates renamed in https://github.com/open-contracting/extension-explorer/pull/67/commits/f254e86f77cf25627685c5dfadf3d5db662db0c7